### ISSUE TITLE
Error on Connect: Command 'mssql.connect' not found

### ISSUE THREAD

**Issue by RodBarnes** [microsoft/vscode-mssql#17867]
- MSSQL Extension Version: 1.9.0
- VSCode Version: 1.85.1
- OS Version: Linux x64 6.5.0-14-generic (LinuxMint 21.3)

Steps to Reproduce:

1. Install SQL Server (mssql) extension
2. Ctrl-Shift-P
3. Select "MS SQL:Connect"

Error displays:
```
Command 'MS SQL:CONNECT' resulted in an error
command 'mssql.connect' not found
```

I have no other extensions installed -- zero.

I found this issue which has been closed #1022, and I've tried several of the things listed in the comments but nothing has worked.  The comment https://github.com/microsoft/vscode-mssql/issues/1022#issuecomment-660531959 is most near to my system and those steps did not fix the issue.

**Comment by kburtram** [comment]
When the command is not found, this typically indicates a problem launching the extension causing the commands to not be registered.  Are you seeing any exceptions or errors in the Help->Developer Tools Console that may indicate what specifically is happening in this case?

**Comment by kburtram** [comment]
Closing based on previous comment.  One thing I notice is that the extension version appears to be old (1.9.0).  Please try with the latest extension release, currently 1.23.0.

**Comment by mukul1729** [comment]
Hello, I faced this same issue but was resolved upon downloading the 1.24 release, Why does the vscode extension marketplace have a 1.9.0 version? Can you check why it doesn't have the updated version?
